plugins/ws.js

```diff
@@ -137,7 +137,16 @@ module.exports = function (opts = {}) {
         return null
       }
 
-      return URL.format({
+      return Array.isArray(resultHost)
+      ? resultHost.map((h) => {
+        return URL.format({
```

Comment:
can we move this into a helper function?

Reply:
@arj03 i refactored a bit. was not sure where to put these helpers exactly so if they should live somewhere else, let me know. the change is to treat all `external` values as an array so if it comes in as a string, put it in an array, this cut down of some duplication.

Reply:
Much better. I think you can actually leave out the function now.

Reply:
ahh! I like that better as well. Thanks :)